i18n/config.json

```diff
@@ -119,6 +119,16 @@
     "enabled": false,
     "default": false
   },
+  {
+    "code": "pt",
+    "localName": "Português",
+    "name": "Português",
+    "langDir": "ltr",
+    "dateFormat": "MM.DD.YYYY",
+    "hrefLang": "pt",
+    "enabled": true,
```

Comment:
No translation language should be enabled before translations are 100% via Crowdin.

Reply:
Do you have knowledge about when this will be ready?

Could you give me a notification about the conclusion of this process?

Reply:
@ovflowd, it is resolved too.